ha/resource/sspl

```diff
@@ -121,13 +121,21 @@ stateful_start() {
         # CRM Error - Should never happen
         log "START (master)..."
         systemctl start sspl-ll
+        while ! systemctl is-active sspl-ll | grep "active"
```

Comment:
you should be checking "failed" status as well and if service fails to start, return proper OCF code.
Applies to code below too. You can add another function as code will be common and call the function from both the places.

Reply:
Addressed comment in the latest patch